platforms/jvm/platform-jvm/src/integTest/groovy/org/gradle/samples/.gitkeepdir

```diff
@@ -0,0 +1 @@
+// For GenerateAutoTestedSamplesTestTask
```

Comment:
❓ Why aren't these git files needed for every project?  Can you also explain why in these comments?